data/json/items/gun/45colt.json

```
@@ -1,4 +1,61 @@
 [
+  {
+    "id": "bond_410",
+    "looks_like": "glock_17",
+    "type": "GUN",
+    "reload_noise_volume": 10,
+    "name": "Bond Arms Derringer",
+    "description": "The Bond Arms Derringer is a series of multi-barrel compact pistols.  Most commonly chambered for .45 Colt, with chambers long enough to accept .410 shotgun shells.",
+    "weight": "567 g",
+    "volume": "500 ml",
+    "price": 41500,
+    "to_hit": -1,
+    "bashing": 2,
+    "material": [ "steel", "wood" ],
+    "symbol": "(",
+    "color": "brown",
+    "ammo": [ "45colt", "410shot" ],
+    "skill": "pistol",
+    "dispersion": 520,
+    "durability": 6,
+    "clip_size": 2,
+    "blackpowder_tolerance": 80,
+    "faults": [ "fault_gun_blackpowder", "fault_gun_dirt" ],
+    "valid_mod_locations": [
+      [ "accessories", 2 ],
+      [ "barrel", 1 ],
+      [ "bore", 1 ],
+      [ "grip", 1 ],
+      [ "mechanism", 2 ],
+      [ "sights mount", 1 ],
+      [ "stock mount", 1 ],
+      [ "underbarrel mount", 1 ]
+    ],
+    "flags": [ "RELOAD_ONE", "NEVER_JAMS", "RELOAD_EJECT" ]
+  },
+  {
+    "id": "colt_lightning",
+    "copy-from": "rifle_manual",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": "Colt Lightning .45 Carbine",
+    "description": "A modern reproduction of a Colt pump-action rifle.  Originally chambered in .44-40, modern versions most commonly use .45 Colt, complementing the Single Action Army as a Cowboy Action Shooting firearm.",
+    "weight": "2722 g",
+    "volume": "2500 ml",
+    "price": 132000,
+    "to_hit": -1,
+    "bashing": 12,
+    "material": [ "steel", "wood" ],
+    "color": "brown",
+    "ammo": "45colt",
+    "ranged_damage": 5,
+    "dispersion": 180,
+    "durability": 7,
+    "clip_size": 10,
+    "blackpowder_tolerance": 56,
+    "reload_noise": "chuk chuk.",
```

```suggestion
    "reload_noise": "chik chik.",
```

Y'know, cos it's smaller.